src/System-Settings-Tests/SystemSettingsPersistenceTest.class.st

```diff
@@ -103,11 +103,12 @@ SystemSettingsPersistenceTest >> testDefaultImageDoesNotStoreAnySetting [
 	| nodesToStore |
 	"An image with all default values should not save any setting.
 	The settings only should be saved if changed from default value"
-
+"MockSettings foo."
```

Comment:
Leftover?

Reply:
I guess, it's been too long.
Thanks for spotting it.